notebooks/5090_download-scenarios.py

```diff
@@ -168,9 +168,14 @@
 
 
 def check_negatives(df):  # noqa : D103
-    # Filter rows where 'unit' is not "CO2"
-    tmp_not_co2 = df.loc[~df.index.get_level_values("unit").str.contains("CO2")]
-
+    # Filter rows where negative values are allowed
+    mask_exclude_from_check = (
+        df.index.get_level_values("variable").str.contains("CO2")
+        | df.index.get_level_values("variable").str.contains("Removal")
```

Comment:
Isn't "Carbon Removal" always supposed to be positive? So I would delete it here.

Reply:
https://github.com/IAMconsortium/common-definitions/blob/dc4de51f613de5e8f2f16b686106720316cfb8e1/definitions/variable/emissions/carbon-removal.yaml#L6

Reply:
Negative values in Carbon Removal would be a problem